.trunk/configs/svgo.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    plugins: [
+        {
+            name: "preset-default",
+            params: {
+                overrides: {
+                    removeViewBox: false, // https://github.com/svg/svgo/issues/1128
```

Comment:
https://github.com/svg/svgo/pull/1461 was merged, so is this `removeViewBox: false` still necessary?
* If it isn't, let's remove the line.
* If it is, then the comment link should point to whatever issue tracks bringing in the updated svgo

(unless I've misinterpreted?)